helper/hostutil/hostinfo.go

```diff
@@ -0,0 +1,81 @@
+package hostutil
+
+import (
+	"time"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/shirou/gopsutil/cpu"
+	"github.com/shirou/gopsutil/disk"
+	"github.com/shirou/gopsutil/host"
+	"github.com/shirou/gopsutil/mem"
+)
+
+// HostInfo holds all the information that gets captured on the host.
+type HostInfo struct {
+	Timestamp time.Time              `json:"timestamp"`
+	CPU       []cpu.InfoStat         `json:"cpu"`
```

Comment:
May be nice to include comments describing what each of these values are. e.g. is this an array of user, system, idle CPU times?

Reply:
The set of values returned/implemented by `gopsutil` depends on the operating system and whether cgo was used when compiling the binary. The set of stats for each of those are pretty long (memory has 35 different values for instance), so instead I provided general comments on the fields as opposed to listing the specific values.